### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by NikolaevaIrina**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,21 +1,42 @@
...
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T236972)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
+
+# XAF - How to dynamically change mask settings based on the current object
+
+This example demonstrates how to change mask settings of a certain editor dynamically, e.g. based on properties of the current object.
+
```
Comment (AndreyKozhevnikov) [inline Readme.md:9]: chrome_pDcpJI0J3C

давай картинку добавим

Comment (NikolaevaIrina) [inline Readme.md:9]:
```suggestion
  
<kbd>![example](./CS/EFCore/DynamicMaskEF/DynamicMaskEF.Module/Images/how-to-dynamically-change-mask-settings.gif)</kbd>
```

File: Readme.md [hunk 2]
```diff
@@ -1,21 +1,44 @@
...
-![](https://img.shields.io/endpoint?url=https://codecentral.devexpress.com/api/v1/VersionRange/128589903/22.2.5%2B)
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T236972)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
+
+# XAF - How to dynamically change mask settings based on the current object
+
+This example demonstrates how to change mask settings of a certain editor dynamically, e.g. based on properties of the current object.
```
Comment (albertov05) [inline Readme.md]:
```suggestion
This example demonstrates how to change mask settings of a certain editor dynamically (for example, based on properties of the current object).
```

File: Readme.md [hunk 3]
```diff
@@ -1,21 +1,44 @@
...
+  
+<kbd>![example](./CS/EFCore/DynamicMaskEF/DynamicMaskEF.Module/Images/how-to-dynamically-change-mask-settings.gif)</kbd>
+
+## Implementation Details
+
+There are two ways to implement this functionality:
+
+1. Implement a `ViewController` that handles events of the current view and changes settings of required editors, as shown in the following topic: [Access the Settings of a Property Editor in a Detail View](https://docs.devexpress.com/eXpressAppFramework/402153/getting-started/in-depth-tutorial-blazor/customize-data-display-and-view-layout/access-editor-settings).
+2. Implement a custom Property Editor (e.g. a descendant of the corresponding built-in property editor) and change settings of its control. For more information, refer to the following help section: [Property Editors](https://docs.devexpress.com/eXpressAppFramework/113097/ui-construction/view-items-and-property-editors/property-editors?p=netframework).
```
Comment (albertov05) [inline Readme.md]: 1. Let's change "e.g." (which technically requires a comma at the end) to "for example" (with a comma at the end).

2.  What do you mean by "and change settings of **its control**"?  So the custom Property Editor has another control within it? Please clarify.

File: Readme.md [hunk 4]
```diff
@@ -1,21 +1,44 @@
...
+
+This example demonstrates the first approach. To implement it, the following classes are added:
+
+* `DemoObject` - a persistent class with `TestString` and `Mask` properties. Mask settings of the `TestString` property are changed based on the value of the `Mask` property.
+* `ChangeMaskControllerBase` - a platform-independent controller that handles events required to update the mask settings at the appropriate time.
+* `WinChangeMaskController` - a WinForms-specific controller that customizes settings of a WinForms control.
+* `BlazorChangeMaskController` - an Blazor-specific controller that customizes settings of a Blazor control.
```
Comment (albertov05) [inline Readme.md]:
```suggestion
* `BlazorChangeMaskController` - a Blazor-specific controller that customizes settings of a Blazor control.
```

File: Readme.md (hunk 3 above)
Comment (NikolaevaIrina) [reply Readme.md]: Hi Albert. Yes, each property editor is represented by the corresponding control in the UI (https://docs.devexpress.com/eXpressAppFramework/113097/ui-construction/view-items-and-property-editors/property-editors?p=net6). Can we rephrase this sentance in the following way?

```suggestion
2. Implement a custom Property Editor (for example, a descendant of the corresponding built-in property editor) and change settings of the associated underlying control. For more information, refer to the following help section: [Property Editors](https://docs.devexpress.com/eXpressAppFramework/113097/ui-construction/view-items-and-property-editors/property-editors?p=netframework).
```

Comment (albertov05) [reply Readme.md]: Thank you, Irina!

**Pull request merged**